Equilobe.DailyReport.Utils/XmlHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Equilobe.DailyReport.Utils
+{
+    public static class XmlHelper
+    {
+        public static T DeserializeXml<T>(string data)
```

Comment:
We don't actually need this. You just need to deserialize into the desired class. 

DeserializeXml<ClassName>(string data) should be enough, no need to copy properties on another new created object.